src/main/java/com/siemens/ct/exi/grammars/EXIContentModelBuilder.java

```diff
@@ -333,8 +336,8 @@ protected SchemaInformedGrammar handleParticle(XSComplexTypeDefinition ctd,
 
 			CMState startState = new CMState(possibleElements, isEnd, state,
 					elementsMaxOccursUnbounded, occurenceInfo);
-			if (DEBUG) {
-				System.out.println("Start = " + startState);
+			if (LOGGER.isDebugEnabled()) {
```

Comment:
I don't think wrapping in `if` statement is necessary since LOGGER.debug will not log anything if DEBUG level is not reached.